### PR TITLE
feat: logger for the WorldstateClient

### PR DESCRIPTION
**Pull request by Mettwasser**
### What did you fix? 
Implements a proper logger instead of the *ahem* shitty `if self._debug:`

---

### Reproduction steps

---

### Considerations
- Does this contain a new dependency? **[No]**
- Does this introduce opinionated data formatting or manual data entry? **[No]**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **[Yes]**
- Have I run the linter? **[Yes/No]**
- Is is a bug fix, feature request, or enhancement? **[Feature/Docs]**

### CODE REVIEW

File: examples/worldstate/custom_logger.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# queries are just another way to get the data to the corresponding objects.
```
Comment (TobiTenno) [inline examples/worldstate/custom_logger.py]: This file should just be in your readme for an example... This doesn't really make sense to me.

**Review by TobiTenno — CHANGES_REQUESTED** [review]

File: examples/worldstate/custom_logger.py (hunk 1 above)
Comment (Mettwasser) [reply examples/worldstate/custom_logger.py]: ..what do you want o.O

Comment (TobiTenno) [reply examples/worldstate/custom_logger.py]: this in the readme

**Review by TobiTenno — CHANGES_REQUESTED** [review]
rebase

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**